docs/prob/_sidebar.md

```diff
@@ -4,21 +4,28 @@
 
 - [Problems](prob/)
 
+  - [/prob/api-client/unexpected-read-error](/prob/api/unexpected-read-error.md)
+  - [/prob/api-client/unexpected-trigger-error](/prob/api/unexpected-trigger-error.md)
+  - [/prob/api-client/unexpected-write-error](/prob/api/unexpected-write-error.md)
   - [/prob/api/internal-server-error](/prob/api/internal-server-error.md)
   - [/prob/api/invalid-param-int](/prob/api/invalid-param-int.md)
-  - [/prob/api/invalid-param-uint](/prob/api/invalid-param-uint.md)
   - [/prob/api/invalid-param](/prob/api/invalid-param.md)
+  - [/prob/api/invalid-param-uint](/prob/api/invalid-param-uint.md)
```

Comment:
This seems a little off. Apparently `sort` only sorts on alphanumerical characters by default. So for the eyes of `sort` the lines are:

```md
probapiinvalidparamintprobapiinvalidparamintmd
probapiinvalidparamuintprobapiinvalidparamuintmd
probapiinvalidparamprobapiinvalidparammd
```

Best option seems to be to just import it into some text editor and let that one sort on all chars as expected, resulting in:

```suggestion
  - [/prob/api/invalid-param-uint](/prob/api/invalid-param-uint.md)
  - [/prob/api/invalid-param](/prob/api/invalid-param.md)
```